### PR TITLE
Fix/webpack

### PR DESCRIPTION
**Pull request by 0x0xyz**
This MR provides multiple refactorings - both on the code side, but also in terms of how the packages are being built and bundled.

### CODE REVIEW

**Review by 3s-jorgematos — APPROVED** [review]
LGTM

**Review by JoseAfonsoThreeSigma — APPROVED** [review]
LGTM but please squash all commits into to avoid polluting master's history

File: src/packages/crypto/webpack.config.js [hunk 1]
```diff
@@ -65,6 +79,23 @@ const baseConfigUmd = {
...
+            presets: [
+              '@babel/preset-env',
+              '@babel/preset-react',
+              '@babel/preset-typescript',
+              '@babel/preset-flow',
+            ],
+          },
+        },
+      },
+      */
```
[the remark this answers is not in the document]
Comment (0x0xyz) [reply src/packages/crypto/webpack.config.js]: removed

File: src/packages/client/webpack.config.js [hunk 2]
```diff
@@ -65,6 +79,23 @@ const baseConfigUmd = {
...
+            presets: [
+              '@babel/preset-env',
+              '@babel/preset-react',
+              '@babel/preset-typescript',
+              '@babel/preset-flow',
+            ],
+          },
+        },
+      },
+      */
```
[the remark this answers is not in the document]
Comment (0x0xyz) [reply src/packages/client/webpack.config.js]: removed

**Pull request merged**